### PR TITLE
Add Address Parser

### PR DESCRIPTION
**Pull request by stdavis**
~This is nowhere close to being finished. Just wanted to open this up and get opinions on the direction that I'm taking...~

I believe that the address parser is to a point of being useable and would like to get this merged. I've still got open issues, but I think that they can go in subsequent PRs. I'd like to get this into users' hands for feedback.

The basic API looks like this:
```python
from sweeper.address_parser import Address

address = Address('123 south main street')
print(address.address_number) #: '123'
print(address.prefix_direction) #: 'S'
```

### CODE REVIEW

**Comment by steveoh** [comment]
Cool, i'll check it out

File: readme.md [hunk 1]
```diff
@@ -10,3 +63,4 @@ fix data
    - `activate sweeper`
 1. install dependencies
    - `conda install -y -f --file requirements.dev.txt`
```
Comment (steveoh) [inline readme.md]: this does not work

**Comment by steveoh** [comment]
i think the setup.py is fouled up also so you can't pip install it from the clone and use it in other projects. None of the install requires match up.

**Comment by stdavis** [comment]
Thanks for the failing test. I made it pass.

The pip stuff looks great to me also. Thanks!

**Pull request merged**